src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.ts

```diff
@@ -39,18 +48,62 @@ class TeamsReportDeviceUsageDistributionUserCountsCommand extends GraphCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
-        cmd.log(res);
+        let content: string = '';
+
+        if (args.options.output && args.options.output.toLowerCase() === 'json') {
+          let reportdata = this.getJsonReport(res);
+          content = JSON.stringify(reportdata);
+
+          if (!args.options.outputFile) {
+            cmd.log(reportdata);
+          }
+        }
+        else {
+          content = res;
+
+          if (!args.options.outputFile) {
+            cmd.log(content);
+          }
+        }
+
+        if (args.options.outputFile) {
+          fs.writeFileSync(args.options.outputFile, content, 'utf8');
+          if (this.verbose) {
+            cmd.log(`File saved to path '${args.options.outputFile}'`);
+          }
+        }
 
         cb();
       }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
-}
+  }
+
+  private getJsonReport(res: string): any {
+    let rows = res.split('\n');
```

Comment:
This and the following two variables should be const

Reply:
Done, fixed this.